rosidl_generator_py/resource/_msg.py.em

```diff
@@ -211,7 +211,7 @@ bound = 2**nbits
 @[      end if]@
 @[  elif field.type.string_upper_bound]@
             ((isinstance(value, str) or isinstance(value, UserString)) and
-             len(value) <= @(field.type.string_upper_bound))
+             len(value.encode('utf-8')) <= @(field.type.string_upper_bound))
```

Comment:
I am not sure atm. I think the design doc needs to be clear in terms of strings / wstrings before I know how this should be implemented.